src/cpu/x64/jit_avx512_core_amx_conv_kernel.hpp

```diff
@@ -50,7 +50,11 @@ struct jit_avx512_core_amx_compute_zp_pbuff_t : public jit_generator_t {
 private:
     jit_conv_conf_t jcp;
 
-    typedef enum { no_last_block, last_ic_block } ic_block_t;
+    enum class ic_block_t {
+        no_last_block,
+        last_ic_block,
```

Comment:
nit: The redundant naming is redundant. Maybe just make the values `no_last` and `last`?

Reply:
Thanks for a suggestion. While it seems reasonable, I'll pass on this one and let CPU team to deal with it in the future if they want to. Names present in multiple different spots, used in a different way (some as just values, others as bits), so I don't introduce inconsistencies in names while there's still inconsistency in use.